opencti-platform/opencti-graphql/config/default.json

```diff
@@ -162,6 +166,10 @@
     "lock_digest_key": "notification_digest_manager_lock",
     "interval": 10000
   },
+  "telemetry_manager": {
+    "enabled": true,
+    "lock_key": "telemetry_manager_lock",
```

Comment:
there is a ',' at the end of line that prevents the backend to start

Reply:
fixed